Tests/DependencyInjection/DoctrineExtensionTest.php

```diff
@@ -676,6 +677,31 @@ public function testAnnotationsBundleMappingDetectionWithVendorNamespace()
         $this->assertEquals('Fixtures\Bundles\Vendor\AnnotationsBundle\Entity', $calls[0][1][1]);
     }
 
+    public function testMessengerIntegration()
+    {
+        if (! interface_exists(MessageBusInterface::class)) {
```

Comment:
I'd argue that it should be a dev-dependency, otherwise it's untested.

Reply:
Oops, I though `symfony/symfony` was used on CI for specific versions. So I expected it to be available in the new build.
The issue with adding it into the `dev-dependencies` (see first commit) is that it'll conflict with lowest dependencies tested for this packages due to the component requirements. So it must be explicitly added on a specific build. WDYT?

Reply:
You can override it for now using `^4.1@beta`, the `@beta` part would be dropped once 4.1 is stable. But yes, it wouldn't allow SF pre-3.4. Since there is only one (or two) CI job to test SF 2.x, I think it'd be better to drop this dependency only in such job.
(Another argument for dropping 2.x soon.)